src/ziggurat/middleware/json.clj

```diff
@@ -0,0 +1,45 @@
+(ns ziggurat.middleware.json
+  "This namespace defines middleware methods for parsing JSON strings.
+   Please see [Ziggurat Middleware](https://github.com/gojek/ziggurat#middleware-in-ziggurat) for more details.
+  "
+  (:require [cheshire.core :refer [parse-string]]
+            [sentry-clj.async :as sentry]
+            [ziggurat.sentry :refer [sentry-reporter]]
+            [ziggurat.metrics :as metrics]))
+
+(defn- deserialize-json
+  [message topic-entity key-fn]
+  (try
+    (parse-string message key-fn)
+    (catch Exception e
+      (let [additional-tags   {:topic_name topic-entity}
```

Comment:
the `topic-entity` that is being passed here, is that a string or a keyword? We want to send a string in the metric right? thats what I was referring to in the previous comment.

Reply:
Okay. Done.